### DISCUSSION TITLE
Get frontmatter for literature notes from Goodreads link

### DISCUSSION THREAD

**Discussion by basilioss** [SilentVoid13/Templater#611] [Templates Showcase]
~~Gist: https://gist.github.com/basilioss/62adab009eeff3b9e043d88ccb74ef82~~

https://github.com/basilioss/obsidian-scrapers

https://user-images.githubusercontent.com/71596800/178814251-1da1a660-7779-4645-8db3-d456f7fe66c8.mp4

**Comment by AB1908** [comment]
Great Scott! This is amazing! If only Goodreads would be as good with their API. I've resorted to using Open Library meanwhile.

**Reply by basilioss** [reply]
What are the advantages of Open Library? I guess Goodreads is more popular and has more books.

**Reply by AB1908** [reply]
OpenLibrary has a public API whereas Goodreads is deprecating theirs IIRC.

**Comment by dissyllabic** [comment]
Hey, this is awesome! Thanks man!

**Reply by userrand** [reply]
I think goodreads does not allow web scraping according to https://law.stackexchange.com/questions/52397/goodreads-app-scraping-quotes-is-it-legal. Not sure what exactly are the consequences if goodreads notices the web scraping.
